src/imagej/dims.py

```diff
@@ -467,3 +475,56 @@ def _to_ijdim(key: str) -> str:
         return ijdims[key]
     else:
         return key
+
+
+def _cal_axis_to_str(key) -> str:
+    """
+    Convert a CalibratedAxis class (e.g. net.imagej.axis.DefaultLinearAxis) to
+    a string.
+    """
+    cal_axis_to_str = {
+        jc.ChapmanRichardsAxis: "ChapmanRichardsAxis",
+        jc.DefaultLinearAxis: "DefaultLinearAxis",
+        jc.EnumeratedAxis: "EnumeratedAxis",
+        jc.ExponentialAxis: "ExponentialAxis",
+        jc.ExponentialRecoveryAxis: "ExponentialRecoveryAxis",
+        jc.GammaVariateAxis: "GammaVariateAxis",
+        jc.GaussianAxis: "GaussianAxis",
+        jc.IdentityAxis: "IdentityAxis",
+        jc.InverseRodbardAxis: "InverseRodbardAxis",
+        jc.LogLinearAxis: "LogLinearAxis",
+        jc.PolynomialAxis: "PolynomialAxis",
+        jc.PowerAxis: "PowerAxis",
+        jc.RodbardAxis: "RodbardAxis",
+    }
+
+    if key.__class__ in cal_axis_to_str:
+        return cal_axis_to_str[key.__class__]
+    else:
+        return "unknown"
```

Comment:
Could be simplified:
`cal_axis_to_str(key.__class__, default="unknown")`

Reply:
I'm no longer using a dict to store the `CalibratedAxis` types. See 56ca2724a7f1a88fa30648d76c43427827a8aa49.